context-free.md

```diff
@@ -2,21 +2,24 @@
 
 ##  Who is this programming language for?
 
+This language appears to be for people who want to create graphics programmatically (with repeated, recursive, random, etc. structures).
```

Comment:
I thought it was particularly helpful that you mentioned repeated, recursive, random, etc. structures. I had not really considered these things when using context-free